src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                + "color='" + color + '\''
+                + "number='" + number + '\''
+                + '}';
+    }
+}
```

Comment:
Class 'Ball' should have an access modifier, preferably 'public' if it's meant to be used outside of its package.